perfdash/metrics-downloader.go

```diff
@@ -42,15 +42,17 @@ type DownloaderOptions struct {
 
 // Downloader that gets data about results from a storage service (GCS) repository.
 type Downloader struct {
-	MetricsBkt MetricsBucket
-	Options    *DownloaderOptions
+	MetricsBkt              MetricsBucket
+	Options                 *DownloaderOptions
+	allowParsersForAllTests bool
 }
 
 // NewDownloader creates a new Downloader.
-func NewDownloader(opt *DownloaderOptions, bkt MetricsBucket) *Downloader {
+func NewDownloader(opt *DownloaderOptions, bkt MetricsBucket, allowAllParsers *bool) *Downloader {
```

Comment:
This should be also bool, it doesn't make sense to allow *bool here, what should we do if caller provides nil?